test/framework/operator.go

```diff
@@ -298,3 +304,88 @@ func CreateNamespacedSubscriptionIfNotExist(namespace string, subscriptionName s
 }
+
+// DeployMongoDBOperatorFromYaml Deploy Kogito Operator from yaml files
+func DeployMongoDBOperatorFromYaml(namespace string) error {
+	GetLogger(namespace).Infof("Deploy MongoDB from yaml files in %s", mongoDBOperatorDeployFilesURI)
+
+	if !infrastructure.IsMongoDBAvailable(kubeClient) {
+		if err := loadResource(namespace, mongoDBOperatorDeployFilesURI+"crds/mongodb.com_mongodb_crd.yaml", &apiextensionsv1beta1.CustomResourceDefinition{}, func(object interface{}) {
+			// Short fix as 'plural' from mongodb is not mongodbs ...
+			// https://github.com/mongodb/mongodb-kubernetes-operator/issues/237
+			crdName := object.(*apiextensionsv1beta1.CustomResourceDefinition).Spec.Names.Plural
+			if !strings.HasSuffix(crdName, "s") {
+				crdName += "s"
+				metadataName := fmt.Sprintf("%s.%s", crdName, object.(*apiextensionsv1beta1.CustomResourceDefinition).Spec.Group)
+
+				GetLogger(namespace).Infof("MongoDB Crd, changing plural from %s to %s", object.(*apiextensionsv1beta1.CustomResourceDefinition).Spec.Names.Plural, crdName)
+				GetLogger(namespace).Infof("MongoDB Crd, changing metadata name from %s to %s", object.(*apiextensionsv1beta1.CustomResourceDefinition).ObjectMeta.Name, metadataName)
+
+				object.(*apiextensionsv1beta1.CustomResourceDefinition).Spec.Names.Plural = crdName
+				object.(*apiextensionsv1beta1.CustomResourceDefinition).ObjectMeta.Name = metadataName
+			}
+		}); err != nil {
+			return err
+		}
+	}
+
+	// TODO: error handling, go lint is screaming about this
+	if err := loadResource(namespace, mongoDBOperatorDeployFilesURI+"service_account.yaml", &corev1.ServiceAccount{}, nil); err != nil {
+		return err
+	}
+	if err := loadResource(namespace, mongoDBOperatorDeployFilesURI+"role.yaml", &rbac.Role{}, nil); err != nil {
+		return err
+	}
+	if err := loadResource(namespace, mongoDBOperatorDeployFilesURI+"role_binding.yaml", &rbac.RoleBinding{}, nil); err != nil {
+		return err
+	}
+
+	// Then deploy operator
+	err := loadResource(namespace, mongoDBOperatorDeployFilesURI+"operator.yaml", &coreapps.Deployment{}, func(object interface{}) {
+		if IsOpenshift() {
+			GetLogger(namespace).Debugf("Setup MANAGED_SECURITY_CONTEXT env in MongoDB operator for Openshift")
+			object.(*coreapps.Deployment).Spec.Template.Spec.Containers[0].Env = append(object.(*coreapps.Deployment).Spec.Template.Spec.Containers[0].Env,
+				corev1.EnvVar{
+					Name:  "MANAGED_SECURITY_CONTEXT",
+					Value: "true",
+				})
+		}
+	})
+	if err != nil {
+		return err
+	}
+
+	// Set correct file to be deployed
+	if IsOpenshift() {
+		// Used to give correct access to pvc/secret
+		// https://github.com/mongodb/mongodb-kubernetes-operator/issues/212#issuecomment-704744307
+		output, err := CreateCommand("oc", "adm", "policy", "add-scc-to-user", "anyuid", "system:serviceaccount:"+namespace+":mongodb-kubernetes-operator").WithLoggerContext(namespace).Sync("add-scc-to-user").Execute()
```

Comment:
Why this command needs to be synchronized?

Reply:
when executed in parallel and almost simultaneously, this creates conflict and only one command will succeed. This allows to wait for execution of the command and avoid failing command execution when parallel

Reply:
ok